core/src/com/unciv/ui/newgamescreen/NewGameScreen.kt

```diff
@@ -206,6 +219,28 @@ class NewGameScreen(
         }).expandX().fillX().row()
     }
 
+    private fun checkConnectionToInternet(): Boolean {
+        return try {
+            val u = URL("https://content.dropboxapi.com")
+            val conn = u.openConnection()
+            conn.connect()
+            true
+        } catch (ex: Throwable) {
+            false
+        }
+    }
+
+    private fun checkConnectionToMultiplayerServer(): Boolean {
+        return try {
+            val u = URL(UncivGame.Current.settings.multiplayerServer)
+            val conn = u.openConnection()
+            conn.connect()
```

Comment:
What does the connect do exactly? Does it try to open the TLS sessions without any HTTP request?
If it does a HTTP request, please consider to request https://serverurl/isalive
Like in [Unciv_server.kt](https://github.com/yairm210/Unciv/blob/e1363d92f9d7bde846218123da90f14e2d66a311/desktop/src/com/unciv/app/desktop/UncivServer.kt ) line 45.